src/uci.cpp

```diff
@@ -314,8 +314,13 @@ string UCI::value(Value v) {
 
   stringstream ss;
 
-  if (abs(v) < VALUE_MATE_IN_MAX_PLY)
+  if (abs(v) < VALUE_TB_WIN_IN_MAX_PLY)
       ss << "cp " << v * 100 / NormalizeToPawnValue;
+  else if (abs(v) < VALUE_MATE_IN_MAX_PLY)
+  {
+      const int ply = VALUE_MATE_IN_MAX_PLY - abs(v);
+      ss << "cp " << (v > 0 ? 10000 - ply : -10000 + ply);
```

Comment:
I would prefer the spelling `(v > 0 ? : 1 : -1) * (10000 - ply)`, which both better shows the essential calculation and minimizes unnecessary duplicate words, especially the meaningful constant `10000`